src/app/code/community/Shopgate/Cloudapi/Helper/Api2/Quote.php

```diff
@@ -142,13 +142,12 @@ function (Mage_Sales_Model_Quote_Address_Total $total) {
             $quote->getTotals()
         );
 
-        if (isset($totals[self::KEY_SHIPPING]) && $totals[self::KEY_SHIPPING]['value']) {
-            $totals[self::KEY_SHIPPING]['value'] = Mage::helper('tax')->getShippingPrice(
-                $totals[self::KEY_SHIPPING]['value'],
-                Mage::helper('tax')->displayShippingPriceIncludingTax() || Mage::helper('tax')->displayShippingBothPrices(),
-                $quote->getShippingAddress(),
-                $quote->getCustomerTaxClassId()
-            );
+        if (isset($totals[self::KEY_SHIPPING], $totals[self::KEY_SHIPPING]['value'])
+            && $totals[self::KEY_SHIPPING]['value']
```

Comment:
may not need to check this as you are not really using this `value` as base for your calculations. I am assuming when $totals[shipping] is set that means some sort of shipping was applied.
A bit of a tricky area as any plugin can rewrite this via their own collector.

Reply:
true that.